sdk/python/kfp_tekton/compiler/_op_to_template.py

```diff
@@ -490,9 +490,15 @@ def _op_to_template(op: BaseOp, enable_artifacts=False):
     if processed_op.pod_annotations or processed_op.pod_labels:
         template.setdefault('metadata', {})  # Tekton change, don't wipe out existing metadata
         if processed_op.pod_annotations:
-            template['metadata']['annotations'] = processed_op.pod_annotations
+            template['metadata']['annotations'] = {
+                sanitize_k8s_name(key, True, True, True, 253): value
```

Comment:
nit: There are some True flags for allowing dot/underscore/slash. Maybe is better to use [python keyword arguments](https://www.w3schools.com/python/gloss_python_function_keyword_arguments.asp) to explicitly show the special characters we are allowing here?

Reply:
I wanted to keep those code blocks as short as possible, but you are right, it is easier to read and prevents errors in the future if we use explicit parameter names